diff --git a/src/main/java/com/artofsolving/jodconverter/DefaultDocumentFormatRegistry.java b/src/main/java/com/artofsolving/jodconverter/DefaultDocumentFormatRegistry.java
index 476f343..a996b21 100644
--- a/src/main/java/com/artofsolving/jodconverter/DefaultDocumentFormatRegistry.java
+++ b/src/main/java/com/artofsolving/jodconverter/DefaultDocumentFormatRegistry.java
@@ -67,11 +67,11 @@ public class DefaultDocumentFormatRegistry extends BasicDocumentFormatRegistry {
 		addDocumentFormat(wpd);
 
 		final DocumentFormat txt = new DocumentFormat("Plain Text", DocumentFamily.TEXT, "text/plain", "txt");
-        // default to "Text (encoded)" UTF8,CRLF to prevent OOo from trying to display the "ASCII Filter Options" dialog
+        // default to "Text (encoded)" UTF8 to prevent OOo from trying to display the "ASCII Filter Options" dialog
         txt.setImportOption("FilterName", "Text (encoded)");
         txt.setImportOption("FilterOptions", "UTF8,CRLF");
 		txt.setExportFilter(DocumentFamily.TEXT, "Text (encoded)");
-		txt.setExportOption(DocumentFamily.TEXT, "FilterOptions", "UTF8,CRLF");
+		txt.setExportOption(DocumentFamily.TEXT, "FilterOptions", "UTF8");
 		addDocumentFormat(txt);
 
 		final DocumentFormat wikitext = new DocumentFormat("MediaWiki wikitext", "text/x-wiki", "wiki");
diff --git a/src/main/resources/com/artofsolving/jodconverter/document-formats.xml b/src/main/resources/com/artofsolving/jodconverter/document-formats.xml
index f4dd3d1..a4cb02e 100644
--- a/src/main/resources/com/artofsolving/jodconverter/document-formats.xml
+++ b/src/main/resources/com/artofsolving/jodconverter/document-formats.xml
@@ -240,7 +240,7 @@
         <map>
           <entry>
             <string>FilterOptions</string>
-            <string>UTF8,CRLF</string>
+            <string>UTF8</string>
           </entry>
           <entry>
             <string>FilterName</string>
